pkg/JuliaInterface/src/convert.c

```diff
@@ -27,23 +28,28 @@ jl_value_t * julia_gap(Obj obj)
     if (obj == False) {
         return jl_false;
     }
+    jl_value_t * result = (jl_value_t *) obj;
+    BEGIN_GAP_SYNC();
     if (TNUM_OBJ(obj) >= FIRST_EXTERNAL_TNUM &&
         CALL_1ARGS(JULIAINTERFACE_IsJuliaWrapper, obj) == True) {
         obj = CALL_1ARGS(JULIAINTERFACE_JuliaPointer, obj);
         if (IS_JULIA_OBJ(obj)) {
-            return GET_JULIA_OBJ(obj);
+            result = GET_JULIA_OBJ(obj);
         }
         // to handle immediate integers, booleans, FFEs, IS_JULIA_FUNC, we
         // use recursion - but only for internal types; this prevents an
         // infinite recursion, e.g. if an object's JuliaPointer points to the
         // object itself
-        if (TNUM_OBJ(obj) < FIRST_EXTERNAL_TNUM)
-            return julia_gap(obj);
-        ErrorMayQuit("JuliaPointer must be a Julia object or an internal GAP "
-                     "object (not a %s)",
-                     (Int)TNAM_OBJ(obj), 0);
+        else if (TNUM_OBJ(obj) < FIRST_EXTERNAL_TNUM) {
+            result = julia_gap(obj);
+        } else {
```

Comment:
```suggestion
        } else
        {
```
otherwise fails the clang-format check

Reply:
Also fixed.